src/astro/sql/operators/sql_decorator.py

```diff
@@ -237,74 +238,36 @@ def _set_schema_if_needed(self):
             )
         self._run_sql_string(schema_statement, {})
 
-    def get_snow_hook(self) -> SnowflakeHook:
-        """
-        Create and return SnowflakeHook.
-        :return: a SnowflakeHook instance.
-        :rtype: SnowflakeHook
-        """
-        return SnowflakeHook(
-            snowflake_conn_id=self.conn_id,
-            warehouse=self.warehouse,
-            database=self.database,
-            role=self.role,
-            schema=self.schema,
-            authenticator=None,
-            session_parameters=None,
-        )
-
-    def get_bigquery_hook(self):
-        return BigQueryHook(
-            use_legacy_sql=False,
-            gcp_conn_id=self.conn_id,
-        )
-
-    def get_sqlite_hook(self):
-        return SqliteHook(
-            sqlite_conn_id=self.conn_id,
-        )
-
-    def get_postgres_hook(self):
-        return PostgresHook(postgres_conn_id=self.conn_id, schema=self.database)
-
     def _run_sql_string(self, sql, parameters):
```

Comment:
I think we don't need  _run_sql_string() any more, can we replace it with -  _run_sql_alchemy_obj.

Reply:
Done!